languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -37493,6 +37493,110 @@ though, and has since been slightly modified)-->
+                  <token>scénette</token>
+                  <token>»</token>
+              </antipattern>
+              <pattern>
+                  <token>scénette</token>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
+              <suggestion suppress_misspelled="yes"><match no="1" regexp_match="scénette" regexp_replace="saynète"/></suggestion>
+              <example correction="saynète">Cette <marker>scénette</marker> est intéressante.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token skip="-1">jours</token>
+                  <marker>
+                      <token>bimestriel</token>
+                  </marker>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
+              <suggestion><match no="2" regexp_match="bimestriel" regexp_replace="bimensuel"/></suggestion>
+              <example correction="bimensuel">Tous les quinze jours, je reçois mon magasine <marker>bimestriel</marker>.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token postag="UNKNOWN"/>
+                  <token inflected="yes">solutionner</token>
+              </antipattern>
+              <antipattern>
+                  <token>«</token>
+                  <token inflected="yes">solutionner</token>
+                  <token>»</token>
+              </antipattern>
+              <pattern>
+                  <token inflected="yes">solutionner</token>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
+              <suggestion><match no="1" regexp_match="solutionner" regexp_replace="résoudre"/></suggestion>
+              <example correction="résoudre">Il a réussi à <marker>solutionner</marker> le problème.</example>
+              <example>Il a réussi avec Galinette <marker>Solutionnée</marker> à résoudre le problème.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token skip="-1">maline</token>
+                  <token>maligne</token>
+              </antipattern>
+              <antipattern>
+                  <token skip="-1">maligne</token>
+                  <token>maline</token>
+              </antipattern>
+              <antipattern>
+                  <token postag="UNKNOWN"/>
+                  <token>maline</token>
+              </antipattern>
+              <antipattern>
+                  <token>maline</token>
+                  <token postag="UNKNOWN"/>
+              </antipattern>
+              <antipattern>
+                  <token postag="[DJ].*" postag_regexp="yes"/>
+                  <token>maline</token>
+              </antipattern>
+              <pattern>
+                  <token case_sensitive="yes">maline</token>
+              </pattern>
+              <message>Une erreur d’orthographe semble être détectée.</message>
```

Comment:
This, too should be more explanatory. "La forme féminine de l'adjectif 'malin' est 'maligne'."

Reply:
All the comments have been taken into account so the rule is ready.